pallets/ddc-staking/src/weights.rs

```diff
@@ -35,153 +35,306 @@ pub trait WeightInfo {
+	// Proof: Balances Locks (max_values: None, max_size: Some(1299), added: 3774, mode: MaxEncodedLen)
 	// Storage: System Account (r:1 w:1)
+	// Proof: System Account (max_values: None, max_size: Some(128), added: 2603, mode: MaxEncodedLen)
 	fn unbond() -> Weight {
-		Weight::from_parts(37_000_000_u64, 0)
+		Weight::from_parts(686_000_000_u64, 0)
 			.saturating_add(T::DbWeight::get().reads(6_u64))
 			.saturating_add(T::DbWeight::get().writes(3_u64))
 	}
 	// Storage: DdcStaking Ledger (r:1 w:1)
+	// Proof Skipped: DdcStaking Ledger (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Providers (r:1 w:0)
+	// Proof Skipped: DdcStaking Providers (max_values: None, max_size: None, mode: Measured)
 	// Storage: Balances Locks (r:1 w:1)
+	// Proof: Balances Locks (max_values: None, max_size: Some(1299), added: 3774, mode: MaxEncodedLen)
 	// Storage: System Account (r:1 w:1)
+	// Proof: System Account (max_values: None, max_size: Some(128), added: 2603, mode: MaxEncodedLen)
 	// Storage: DdcStaking LeavingStorages (r:1 w:0)
+	// Proof Skipped: DdcStaking LeavingStorages (max_values: None, max_size: None, mode: Measured)
 	fn withdraw_unbonded() -> Weight {
-		Weight::from_parts(33_000_000_u64, 0)
+		Weight::from_parts(565_000_000_u64, 0)
 			.saturating_add(T::DbWeight::get().reads(5_u64))
 			.saturating_add(T::DbWeight::get().writes(3_u64))
 	}
 	// Storage: DdcClusters Clusters (r:1 w:0)
+	// Proof Skipped: DdcClusters Clusters (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Ledger (r:1 w:0)
+	// Proof Skipped: DdcStaking Ledger (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcClusters ClustersGovParams (r:1 w:0)
+	// Proof Skipped: DdcClusters ClustersGovParams (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Providers (r:1 w:0)
+	// Proof Skipped: DdcStaking Providers (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Storages (r:1 w:1)
+	// Proof Skipped: DdcStaking Storages (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking LeavingStorages (r:1 w:0)
+	// Proof Skipped: DdcStaking LeavingStorages (max_values: None, max_size: None, mode: Measured)
 	fn store() -> Weight {
-		Weight::from_parts(28_000_000_u64, 0)
+		Weight::from_parts(471_000_000_u64, 0)
 			.saturating_add(T::DbWeight::get().reads(6_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 	}
 	// Storage: DdcStaking Ledger (r:1 w:1)
+	// Proof Skipped: DdcStaking Ledger (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcStaking Storages (r:1 w:1)
+	// Proof Skipped: DdcStaking Storages (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcClusters ClustersGovParams (r:1 w:0)
+	// Proof Skipped: DdcClusters ClustersGovParams (max_values: None, max_size: None, mode: Measured)
 	fn chill() -> Weight {
-		Weight::from_parts(28_000_000_u64, 0)
+		Weight::from_parts(394_000_000_u64, 0)
```

Comment:
The weights were taken from Rakan's PR and was executed on a dedicated benchmarking machine